packages/@react-native-windows/cli/src/index.ts

```diff
@@ -6,6 +6,48 @@
 
 import * as fs from 'fs';
 import * as path from 'path';
+import * as findUp from 'find-up';
+import {randomBytes} from 'crypto';
+import * as appInsights from 'applicationinsights';
+
+appInsights.setup('795006ca-cf54-40ee-8bc6-03deb91401c3');
```

Comment:
Can you share (internally) on how to access the telemetry?

Reply:
I'll put a link in the OneNote notebook